examples/custom-browser-lib/browser/browserEntry.js

```diff
@@ -0,0 +1,10 @@
+import browserConfig from '@brillout/browser-config';
+
+import './jquery-global'; // see https://stackoverflow.com/a/39820703/1855917
```

Comment:
I've a slight preference for `import './jquery-global.js'` although I'm ok with `import './jquery-global'`. (Because then I know just from reading that it's JS code and not e.g. CSS that is being loaded)

Reply:
haha ok, will fix, I've just seen a talk from Ryan Dahl where he said it's one of his bigger regrets to have made the extension optional here ;)

Reply:
^^